core/tests/performance_framework/perf_services.py

```diff
@@ -271,18 +288,39 @@ def _setup_driver(self, proxy=None, use_proxy=False):
         return driver
 
     def _add_cookie(self, driver):
+        """Adds a cookie to the current session.
+
+        Args:
+            driver: driver. The WebDriver of the current session.
+        """
         if self.dev_appserver_login_cookie:
             driver.get('%s%s' % (self.BASE_URL, feconf.ROBOTS_TXT_URL))
             driver.add_cookie(self.dev_appserver_login_cookie)
 
     def _stop_proxy_server(self, server):
+        """Stops the proxy server.
+
+        Args:
+            server: server. The browsermobproxy server to be stopped.
+        """
         server.stop()
 
     def _stop_driver(self, driver):
+        """Quits the driver and close every associated window.
+
+        Args:
+            driver: driver. The WebDriver of the current session.
+        """
         driver.quit()
 
     def _is_current_user_logged_in(self, driver):
-        """Checks whether a user is already logged in."""
+        """Checks whether a user is already logged in.
+
+        Args:
+            driver: driver. The WebDriver of the current session.
+        Returns:
```

Comment:
Add blank line before it

Reply:
@vibhor98  Isn't it already there? Pardon If I'm missing something

Reply:
I mean before `Returns` also.